.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,6 +5,10 @@
 * [ ] At least one test case is included for this feature or bug fix
 * [ ] Documentation was added or is not needed
 
```

Comment:
Do we need this line?

Reply:
I wanted to keep the empty line that was originally there. Shall we erase it?

Reply:
I think it's okay to remove it since you added line 11.
Honestly, I don't know why we have this empty line though...

Reply:
I removed it.